index.html

```diff
@@ -1884,89 +1917,108 @@ <h2>The <dfn>NDEFRecordType</dfn> string</h2>
       <td><dfn>"`empty`"</dfn></td>
       <td><i>unused</i></td>
       <td><i>unused</i></td>
-      <td><a>Empty record</a></td>
+      <td><a>Empty record</a> (TNF=0)</td>
     </tr>
     <tr>
       <td><dfn>"`text`"</dfn></td>
       <td><i>unused</i></td>
       <td>{{DOMString}}</td>
-      <td><a>Well-known type</a> record with type "`T`"</td>
+      <td><a>Well-known type</a> (TNF=1) record with TYPE="`T`"</td>
     </tr>
     <tr>
       <td><dfn>"`url`"</dfn></td>
       <td><i>unused</i></td>
       <td>{{DOMString}}</td>
-      <td><a>Well-known type</a> record with type "`U`"</td>
+      <td><a>Well-known type</a> (TNF=1) record with TYPE="`U`"</td>
     </tr>
     <tr>
       <td><dfn>"`smart-poster`"</dfn></td>
       <td><i>unused</i></td>
       <td>{{NDEFMessage}}</td>
-      <td><a>Well-known type</a> record with type "`Sp`"</td>
+      <td><a>Well-known type</a> (TNF=1) record with TYPE="`Sp`"</td>
     </tr>
     <tr>
       <td><dfn>"`json`"</dfn></td>
       <td><a>JSON MIME type</a></td>
       <td>
         [= JSON type =]
       </td>
-      <td><a>MIME type record</a> with type equal to
-        <a>MIME type</a>.
+      <td><a>MIME type record</a> (TNF=2) with TYPE= [= JSON type =].
       </td>
     </tr>
     <tr>
       <td><dfn>"`opaque`"</dfn></td>
-      <td><a>MIME type</a></td>
+      <td>[= MIME type =]</a></td>
       <td>{{ArrayBuffer}}</td>
-      <td><a>MIME type record</a></td>
+      <td><a>MIME type record</a> (TNF=2) with TYPE= [= MIME type =]</td>
```

Comment:
with the <a>TYPE field</a> equal to

Reply:
would be too long in a table... I could link them, though